.github/workflows/testing.yml

```diff
@@ -58,6 +58,28 @@ jobs:
       with:
         path: example_valid_py39
 
+  run_action_with_plugin:
+    name: Test run action (plugins)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./
+        with:
+          plugins: flake8-bugbear
```

Comment:
I think we should set the `path` here to `example_bugbear`. Means there are less repeated warnings, and easier for me to check if a PR breaks anything :)

Reply:
ouch, i actually thought of that and then totally forgot. pushing